.github/workflows/release.yml

```
@@ -124,8 +124,19 @@ jobs:
-          # Collect changelog from @connectum/core as the primary changelog
-          CHANGELOG=$(awk "/^## ${VERSION}/{found=1; next} /^## [0-9]/{if(found) exit} found" packages/core/CHANGELOG.md)
+          # Collect changelog from all packages that have entries for this version
+          CHANGELOG=""
+          for pkg_dir in packages/*/; do
+            changelog_file="${pkg_dir}CHANGELOG.md"
+            [ -f "$changelog_file" ] || continue
+            pkg_name=$(jq -r '.name' "${pkg_dir}package.json")
+            section=$(awk "/^## ${VERSION}/{found=1; next} /^## [0-9]/{if(found) exit} found" "$changelog_file")
+            if [ -n "$section" ]; then
+              CHANGELOG="${CHANGELOG}## ${pkg_name}@${VERSION}
+${section}
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)